### ISSUE TITLE
Administratori dicționare

### ISSUE THREAD

**Issue by CatalinFrancu** [dexonline/dexonline#299]
Deja numărul dicționarelor a crescut, trebuie să diseminăm autoritatea asupra dicționarelor către mai multe persoane (de regulă cele care au introdus cea mai mare parte a dicționarului).

Soluția este să adăugăm un cîmp nou la surse (adminId->User.userId), iar în UI să afișăm această informație (probabil)numai moderatorilor.

Eventual să adăugăm o medalie (administrator dicționar)

**Comment by cristiroma** [comment]
Salut, se poate să mă ocup eu de acest ticket? Mulțumesc.

**Comment by CatalinFrancu** [comment]
Absolut, consideră-te asignat. :-) Pare că GitHub nu mă lasă să te asignez dacă nu ești participant la proiect. Vrei să te adaug ca participant?

**Comment by cristiroma** [comment]
Da, mulțumesc!

**Comment by CatalinFrancu** [comment]
Mă asignez și pe mine temporar ca să trec prin biblioteca de acasă și să mă asignez la toate pe care le am. De adus la zi și fișierul „dicționare fizice” (hehe) din Google Drive.

**Comment by CatalinFrancu** [comment]
Misiunea mea s-a încheiat, mă dezabonez. :-) @raduborza, îl pot închide?

**Comment by raduborza** [comment]
Nu, nu-l închide, că acum începe misiunea mea (mi-l asignez mie)
Dicționare care să nu fie în surse n-ai? :)
